### PR TITLE
Contributing.md: Mention UTF-8 and remove the Tolken references

### PR DESCRIPTION
**Pull request by stevecotton**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -51,13 +58,14 @@ public:
 
     // Use leading commas in the ctor list
     // Use the T& foo or T* foo reference and pointer styles, not T &foo or T *foo.
+    // Use the keyword explicit for single-argument constructors
```
Comment (jostephd) [inline .github/CONTRIBUTING.md]: Suggest to quote `explicit`.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -72,18 +80,30 @@ public:
...
-    void exclaim(const std::string& speaker) const
+    t_string exclaim(const std::string& speaker_id) const
     {
-        std::cerr << speaker << ": They're taking the Hobbits to Isengard!" << std::endl;
+        if (how_far_to_destination_ < 100) {
+            if (speaker_id == "signboard") {
+                return _("Oldwood — enter at own risk");
+            } else {
+                // TRANSLATORS: The lake is the small underground one in S06 Temple in the Deep
+                return _("Hmm, someone has written underneath “People who push to master without making pull requests should be thrown in the lake.”");
```
Comment (jostephd) [inline .github/CONTRIBUTING.md]: We don't require pull requests, and in any case I don't think the joke about throwing people in the lake works. I do like the example use of `TRANSLATORS:` and quotes, though.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -72,18 +80,30 @@ public:
...
+     * This returns a translatable string, using gettext's _ function.
+     *
+     * @param speaker_id        The person speaking
      */
-    void exclaim(const std::string& speaker) const
+    t_string exclaim(const std::string& speaker_id) const
     {
-        std::cerr << speaker << ": They're taking the Hobbits to Isengard!" << std::endl;
+        if (how_far_to_destination_ < 100) {
+            if (speaker_id == "signboard") {
```
Comment (jostephd) [inline .github/CONTRIBUTING.md]: No space after `if`.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (stevecotton) [reply .github/CONTRIBUTING.md]: I agree, it was excessive. Will replace with a more in-universe example.

**Pull request merged**